app/models/profile.rb

```diff
@@ -27,14 +86,5 @@ def self.create_from_proofer_applicant(applicant, user)
   end
   # rubocop:enable MethodLength
 
-  def activate
-    transaction do
-      Profile.where('user_id=?', user_id).update_all(active: false)
-      update!(active: true, activated_at: Time.zone.now)
-    end
-  end
-
-  def verified?
-    verified_at.present?
-  end
+  private_class_method :pii_from_applicant
 end
```

Comment:
Create separate classes for these methods? One of the design patterns we've agreed to follow on this project is to keep models lean and to follow SRP. Methods such as `verified?` belong in a Decorator class, encryption and decryption of PII should go in a separate class, etc.